cmd/config.go

```diff
@@ -215,11 +215,20 @@ func writeAppConfig(path string, appConfig *flyctl.AppConfig) error {
 	return nil
 }
 
-func saveAppV2Config(ctx context.Context, apiClient *api.Client, appCompact *api.AppCompact, path string) error {
-	appConfig, err := getAppV2ConfigFromReleases(ctx, apiClient, appCompact.Name)
+func GetRemoteAppV2Config(ctx context.Context, apiClient *api.Client, appCompact *api.AppCompact) (appConfig *appv2.Config, err error) {
```

Comment:
Anything in this file is legacy, if it deserves to survive it should be moved somewhere else

Reply:
Do you mind if i ask where it should be moved? Maybe `internal/config/config.go`?

Reply:
Oh, that is another usual confusion, we have to rename things :)

`internal/config/*` is for flyctl configuration (`~/.fly/*` files), not app configuration (aka fly.toml). 

under `internal/appv2/*`, i.e: `internal/appv2/remoteconfig.go`

Reply:
don't worry about moving stuff around in this pr. it's easier to review changes without code moving files. we can make a separate pr for moving stuff around.